packages/vscode-extension/package.nls.json

```diff
@@ -468,7 +468,7 @@
     "teamstoolkit.chatParticipants.nextStep.whatsNext": "What should I do next?",
     "teamstoolkit.chatParticipants.create.sample": "Scaffold this sample",
     "teamstoolkit.chatParticipants.create.template": "Create this template",
-    "teamstoolkit.chatParticipants.create.tooGeneric": "Your app description is too generic. To find relevant templates or samples, give specific details of your app's capabilities or technologies.\n\nE.g. Instead of saying 'create a chat bot', you could specify 'create a chat bot that answers FAQs for customer support.'",
+    "teamstoolkit.chatParticipants.create.tooGeneric": "Your app description is too generic. To find relevant templates or samples, give specific details of your app's capabilities or technologies.\n\nE.g. Instead of saying 'create a bot', you could specify 'create a bot template' or 'create a notification bot that sends user about stock updates'.",
```

Comment:
Proposed: 'create a notification bot that sends user the stock updates.

Reply:
Thanks, updated.